config/config.go

```diff
@@ -33,7 +33,7 @@ const (
 	// BaseAssetKey is the default asset hash to be used as base asset for all markets. Default is LBTC
 	BaseAssetKey = "BASE_ASSET"
 	// CrawlIntervalKey is the interval in milliseconds to be used when watching the blockchain via the explorer
-	CrawlIntervalKey = "CRAWL_INTERVAL"
+	CrawlIntervalKey = "CRAWL_INTERVAL_IN_MILLISECS"
```

Comment:
I prefer to have shorter ENV variable here. The GoDoc is the place where we point the users to know about the EN vars, we already specify is in milliseconds there